public/js/templates/configpageTemplate/mapProfiles.hbs

```diff
@@ -0,0 +1,81 @@
+<div class="alert alert-info" id="save-alert" role="alert">Saved Configurations!</div>
+<div class="navmenu-brand" href="#">
+  <div class="row profile-tab">
+      Profiles
+    <div class="add-save">
+      {{#if currentDevice}}
+        <button type="button" class="btn btn-warning" id="save-profile">
+          <i class="glyphicon glyphicon-floppy-save"></i>
+        </button>
+        <button type="button" class="btn btn-info" id="add-profile">
+          <i class="glyphicon glyphicon-plus"></i>
+        </button>
+      {{/if}}
+    </div>
+  </div>
+</div>
+<div class="panel-group" id="config-profile-list" role="tablist" aria-multiselectable="false">
+  {{#if currentDevice}}
+    {{#each currentDevice.mapping_profiles}}
+        <div class="panel panel-default">
+          <div class="panel-heading" role="tab" id="hd-{{this.cid}}">
+            <div class= "row">
+              <div class="col-md-6">
+                <h4 class="panel-title profile-title">
+                  {{this.attributes.name}}
+                </h4>
+              </div>
+              <div class="col-md-6 profile-buttons">
+                <a role="button" data-toggle="collapse" data-parent="#config-profile-list" href="#col-{{this.cid}}" aria-expanded="false" aria-controls="col-{{this.cid}}">
```

Comment:
The whole aria- related stuff is for screen readers and accessibility related stuff IIRC. I think we can get rid of all those in all templates (since ours is a visual app and screen readers aren't going to achieve anything here)